activerecord/test/cases/calculations_test.rb

```diff
@@ -17,14 +18,6 @@
 require "models/rating"
 require "models/post"
 
-class NumericData < ActiveRecord::Base
-  self.table_name = "numeric_data"
-
-  attribute :world_population, :integer
-  attribute :my_house_population, :integer
-  attribute :atoms_in_universe, :integer
-end
```

Comment:
Yes, this was added for prevent deprecation message in #15295.
But the deprecation message was removed in #15374.
We do not see deprecation message even if these line are removed.